android/app/src/main/java/com/zulipmobile/notifications/FcmMessage.kt

```diff
@@ -42,8 +42,18 @@ data class Sender(
  * Data identifying where a Zulip message was sent.
  */
 sealed class Recipient {
-    /** A 1:1 private message.  Must have been sent to this user, so nothing more to say. */
-    object Pm : Recipient()
+    /**
+     * A 1:1 private message.  Must be identified by the participants in the
+     * private conversation, i.e. who sent it and who received it.
+     */
+    data class Pm (val senderIdString: String, val userIdString: String ) : Recipient() {
+        fun getPmParticipantIdsString() =
```

Comment:
(There are other comments I would make on this code if we were going to add this logic. But because it's out of scope I don't want to spend your time on them.)

Reply:
Well, very embarrassing to submit a request that won't even compile. I apologize. 
  
As you have probably realized I was confused.  I thought it was normal to receive notifications for both sending and receiving PMs.  But actually I was logged on to both users.  So, this change is a mistake and is unnecessary.

Reply:
> I thought it was normal to receive notifications for both sending and receiving PMs. But actually I was logged on to both users.

Ahh, I see how that happened. Yeah, we never send you a notification for a message you yourself sent.